src/Directory.Build.targets

```diff
@@ -0,0 +1,19 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)../, $(_DirectoryBuildTargetsFile)))/$(_DirectoryBuildTargetsFile)"
+    Condition=" '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)../, $(_DirectoryBuildTargetsFile)))' != '' " />
+
+  <PropertyGroup>
+    <!-- FileVersionRevision is expected to be set by the CI. -->
+    <FileVersion Condition="'$(FileVersionRevision)' != ''">$(VersionPrefix).$(FileVersionRevision)</FileVersion>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IncludeShared)' != 'false'">
+    <Compile Include="$(SrcRoot)Shared/**/*.cs" LinkBase="Shared" Visible="false" />
+    <EmbeddedResource Include="$(SrcRoot)Shared/**/*.resx" LinkBase="Shared" Visible="false">
```

Comment:
Do we have resource files or is this for just in case we add them later?

Reply:
In case we add them later. I included this because the extra logic here is essential for making shared resx files work - didn't want to lose that. But we should be moving to resx files eventually for localization.